### PR TITLE
dsm updates

### PR DESCRIPTION
**Pull request by cswatt**
### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/data_streams/_index.md [hunk 1]
```diff
@@ -13,170 +17,21 @@ Data Streams Monitoring provides a standardized method for teams to understand a
 * Pinpoint faulty producers, consumers or queues, then pivot to related logs or clusters to troubleshoot faster.
 * Prevent cascading delays by equipping service owners to stop backed up events from overwhelming downstream services.
 
-## Setup
+### Setup
 
-{{< programming-lang-wrapper langs="java,go,dotnet" >}}
```
Comment (Hesperide) [inline content/en/data_streams/_index.md:18]: Suggestion, maybe we move this to the end of the overview section? It feels weird to have the table right at the start of setup, it would probably be nicer to have the runtime buttons directly.

File: content/en/data_streams/_index.md [hunk 2]
```diff
@@ -13,170 +17,21 @@ Data Streams Monitoring provides a standardized method for teams to understand a
...
+| Runtime | Supported technologies |
+|---|----|
+| Java | Kafka (self-hosted, Amazon MSK), RabbitMQ, HTTP, gRPC |
+| Go | All (through [manual instrumentation][1]) |
+| .NET | Kafka (self-hosted, Amazon MSK) |
+| Other | See [Agent installation requirements][2] |
```
Comment (Hesperide) [inline content/en/data_streams/_index.md:27]: Maybe we remove 'Other' for now, and we can do this in a separate PR? Essentially, we want 'Other' to link to public docs that cover something like what is in this Google Doc: https://docs.google.com/document/d/1tlZvdEZPEgV-8-645ZV6Ijhs-t-C7m4F0IBAYBrluQo/edit

**Review by Hesperide — APPROVED** [review]

File: content/en/data_streams/_index.md [hunk 3]
```diff
@@ -1,182 +1,67 @@
...
-You can add an additional dimension to end-to-end latency metrics with the `event_type` tag:
+High lag on a consuming service, increased resource use on a Kafka broker and increased RabbitMQ queue size are frequently explained by changes in the way adjacent services are producing to or consuming from these entities.
 
-```go
-_, ctx = datastreams.SetCheckpoint(ctx, "type:internal", "event_type:sell")
-```
+{{< img src="data_streams/data_streams_throughput_tab.png" alt="Datadog Data Streams Monitoring" style="width:100%;" >}}
 
-You only need to add the `event_type` tag for the first service in each pathway. High-cardinality data (such as request IDs or hosts) are not supported as values for the `event_type` tag.
+You can use the **Throughput** tab on any service or queue in Data Streams Monitoring to quickly detect changes in throughput, and which upstream or downstream service these changes originate from. Once the [Service Catalog][2] is configured, you can immediately pivot to the corresponding team's Slack channel or on-call engineer.
```
Comment (alai97) [inline content/en/data_streams/_index.md:60]:
```suggestion
Click on the **Throughput** tab on any service or queue in Data Streams Monitoring to quickly detect changes in throughput, and which upstream or downstream service these changes originate from. Once the [Service Catalog][2] is configured, you can immediately pivot to the corresponding team's Slack channel or on-call engineer.
```

File: content/en/data_streams/dotnet.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    - link: '/data_streams/glossary/'
+      tag: 'Documentation'
+      text: 'Data Streams Glossary'
+---
+
+### Prerequisites
+
+To start with Data Streams Monitoring, you need recent versions of the Datadog Agent and .NET libraries:
+* [Datadog Agent v7.34.0+][1]
+* .NET Tracer v2.17.0+ ([.NET Core][2], [.NET Framework][3])
```
Comment (alai97) [inline content/en/data_streams/dotnet.md:14]:
```suggestion
* [Datadog Agent v7.34.0 or later][1]
* .NET Tracer v2.17.0 or later ([.NET Core][2], [.NET Framework][3])
```

File: content/en/data_streams/dotnet.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+### Installation
+
+.NET uses auto-instrumentation to inject and extract additional metadata required by Data Streams Monitoring for measuring end-to-end latencies and the relationship between queues and services. To enable Data Streams Monitoring, set the environment variable `DD_DATA_STREAMS_ENABLED` to `true` on services sending messages to (or consuming messages from) Kafka.
```
Comment (alai97) [inline content/en/data_streams/dotnet.md:18]:
```suggestion
.NET uses auto-instrumentation to inject and extract additional metadata required by Data Streams Monitoring for measuring end-to-end latencies and the relationship between queues and services. To enable Data Streams Monitoring, set the `DD_DATA_STREAMS_ENABLED` environment variable to `true` on services sending messages to (or consuming messages from) Kafka.
```

File: content/en/data_streams/go.md [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+    - link: '/data_streams/glossary/'
+      tag: 'Documentation'
+      text: 'Data Streams Glossary'
+---
+
+### Prerequisites
+
+To start with Data Streams Monitoring, you need recent versions of the Datadog Agent and Data Streams Monitoring libraries:
+* [Datadog Agent v7.34.0+][1]
+* [Data Streams Library v0.2+][2]
```
Comment (alai97) [inline content/en/data_streams/go.md:14]:
```suggestion
* [Datadog Agent v7.34.0 or later][1]
* [Data Streams Library v0.2 or later][2]
```

File: content/en/data_streams/go.md [hunk 7]
```diff
@@ -0,0 +1,117 @@
...
+
+### Installation
+
+Initiate a Data Streams pathway with `datastreams.Start()` at the start of your pipeline. Then, two types of instrumentation are available:
```
Comment (alai97) [inline content/en/data_streams/go.md:18]:
```suggestion
Initiate a Data Streams pathway with `datastreams.Start()` at the start of your pipeline. 

Two types of instrumentation are available:
```

File: content/en/data_streams/go.md [hunk 8]
```diff
@@ -0,0 +1,117 @@
...
+- Instrumentation for Kafka-based workloads
+- Custom instrumentation for any other queuing technology or protocol
+
+<div class="alert alert-info">The default Trace Agent URL is <code>localhost:8126</code>. If this is different for your application, use the option <code>datastreams.Start(datastreams.WithAgentAddr("notlocalhost:8126"))</code>.</div>
```
Comment (alai97) [inline content/en/data_streams/go.md:22]:
```suggestion
<div class="alert alert-info">The default Trace Agent URL is <code>localhost:8126</code>. If this is different for your application, use the <code>datastreams.Start(datastreams.WithAgentAddr("notlocalhost:8126"))</code> option.</div>
```

File: content/en/data_streams/go.md [hunk 9]
```diff
@@ -0,0 +1,117 @@
...
+...
+
+ddkafka.TraceKafkaProduce(mergedContext, &producedMsg)
+```
+
+### Manual instrumentation
+
+You can also use manual instrumentation.
+
+For example, in HTTP, you can propagate the pathway with HTTP headers.
```
Comment (alai97) [inline content/en/data_streams/go.md:75]:
```suggestion
You can also use manual instrumentation. For example, in HTTP, you can propagate the pathway with HTTP headers.
```

File: content/en/data_streams/java.md [hunk 10]
```diff
@@ -0,0 +1,40 @@
...
+    - link: '/tracing/service_catalog/'
+      tag: 'Documentation'
+      text: 'Service Catalog'
+---
+
+### Prerequisites
+
+To start with Data Streams Monitoring, you need recent versions of the Datadog Agent and Java libraries:
+* [Datadog Agent v7.34.0+][1]
+* [APM enabled with the Java Agent v0.105+][2]
```
Comment (alai97) [inline content/en/data_streams/java.md:17]:
```suggestion
* [Datadog Agent v7.34.0 or later][1]
* [APM enabled with the Java Agent v0.105 or later][2]
```

File: content/en/data_streams/java.md [hunk 11]
```diff
@@ -0,0 +1,40 @@
...
+
+### Installation
+
+Java uses auto-instrumentation to inject and extract additional metadata required by Data Streams Monitoring for measuring end-to-end latencies and the relationship between queues and services. To enable Data Streams Monitoring, set the environment variable `DD_DATA_STREAMS_ENABLED` to `true` on services sending messages to (or consuming messages from) Kafka or RabbitMQ.
```
Comment (alai97) [inline content/en/data_streams/java.md:21]:
```suggestion
Java uses auto-instrumentation to inject and extract additional metadata required by Data Streams Monitoring for measuring end-to-end latencies and the relationship between queues and services. To enable Data Streams Monitoring, set the `DD_DATA_STREAMS_ENABLED` environment variable to `true` on services sending messages to (or consuming messages from) Kafka or RabbitMQ.
```

File: content/en/data_streams/java.md [hunk 12]
```diff
@@ -0,0 +1,40 @@
...
+
+For example:
+```yaml
+environment:
+  - DD_DATA_STREAMS_ENABLED: "true"
+```
+
+As an alternative, you can instead set the system property `-Ddd.data.streams.enabled=true` by running the following when you start your Java application:
```
Comment (alai97) [inline content/en/data_streams/java.md:29]:
```suggestion
As an alternative, you can set the `-Ddd.data.streams.enabled=true` system property by running the following when you start your Java application:
```

**Review by alai97 — CHANGES_REQUESTED** [review]
Some copy suggestions!

**Review by alai97 — APPROVED** [review]
